### ISSUE TITLE
Bug with custom center two popups 

### ISSUE THREAD

**Issue by KavinduDissanayake** [Mijick/PopupView#62]
## Issue Description
When attempting to `.showAndStack()` two separate center popups with custom views at the same time, the background view always adopts a gray color. This behavior occurs despite different configurations intended for the popups. Additionally, an issue with tap gestures is present when a bottom sheet with a `TabView` is used; the button tap does not work upon the first appearance.

### Steps to Reproduce
1. Implement `.showAndStack()` with two separate center popups with custom views.
2. Observe that the background view is gray for both popups.
3. Implement a bottom sheet with a `TabView`.
4. Notice that the button within the `TabView` does not work on the first appearance.

### Expected Behavior
The background view should not default to gray when stacking popups, and all user interactions, such as tap gestures on buttons within a `TabView`, should be responsive upon the first appearance.

### Screenshots
- iPhone 15 Example:
![iPhone 15 Example](https://github.com/Mijick/PopupView/assets/51842810/4b460f17-d4c3-4544-9750-551f64bb2496)

- iPhone 15 Pro Example:
![iPhone 15 Pro Example](https://github.com/Mijick/PopupView/assets/51842810/2e37d6c2-cf70-46a4-9d06-af622372c29e)

- Screenshot of Issue:
![Issue Screenshot](https://github.com/Mijick/PopupView/assets/51842810/1634ae0c-683a-4bf6-ae1b-13da8d7e3670)

### Code Snippet
```swift
struct BreakDownBottomSheetView: BottomPopup {

    func configurePopup(popup: BottomPopupConfig) -> BottomPopupConfig {
        popup
            .tapOutsideToDismiss(true)
            .dragGestureEnabled(true)
            .contentIgnoresSafeArea(true)
    }

    func createContent() -> some View {
        VStack(spacing: 15){
            TabView(selection: $index) {
                ForEach(yourDataArray, id: \.self) { item in
                    Button(action: {
                        // Button action here
                        // Doesn't work on first appearance
                    }) {
                        Text("Button for \(item)")
                    }
                    .tabItem {
                        Image(systemName: "some-system-image")
                        Text("Tab \(item)")
                    }
                    .tag(item)
                }
            }
            .tabViewStyle(.page(indexDisplayMode: .never))
            .indexViewStyle(PageIndexViewStyle(backgroundDisplayMode: .never))
        }
    }
}

BreakDownBottomSheetView()
    .showAndStack()

**Comment by KavinduDissanayake** [comment]
@FulcrumOne  any update on this issue? cc @alina-petrovska

**Comment by KavinduDissanayake** [comment]
@FulcrumOne  Currently, i'm using your library for my project so waiting for your response. Thanks

**Comment by FulcrumOne** [comment]
@KavinduDissanayake, sorry but we have a lot of things to do at work. I'll back to you this weekend

**Comment by KavinduDissanayake** [comment]
No worries, Thanks for your response cc @FulcrumOne

**Comment by FulcrumOne** [comment]
Hey, @KavinduDissanayake

I'm not sure if I understood you correctly, but the first issue (with TabView button not responding) should be fixed with version 2.2.1.

As for the next ones - could you share some screenshots of the app so I can understand what the problems are? Thanks and have a great weekend!

**Comment by KavinduDissanayake** [comment]
More info : 
![282695723-2e37d6c2-cf70-46a4-9d06-af622372c29e](https://github.com/Mijick/PopupView/assets/51842810/27384ed9-c783-44ea-a7f4-281c3f1fcd77)

**Comment by FulcrumOne** [comment]
@KavinduDissanayake, thanks, now I understand your point.

Unfortunately, when I designed the PopupView, the behaviour you described was not the expected one (I wanted to achieve that the popups of different types can be visible and clickable at the same time). 
Nevertheless, I think that the change you requested is possible to achieve by modifying the `PopupView` class from the library. I'll think how I can add this change to the whole system and get back to you.

**Comment by KavinduDissanayake** [comment]
## TAP ISSUE 
I double check every things , may be problem with area that allows for tap. without any modifier button tap work . But When I add any custom view. tap gesture again not work.

```python
struct BreakDownBottomSheetView: BottomPopup {
    // Adding a sample array of strings

    func configurePopup(popup: BottomPopupConfig) -> BottomPopupConfig {
        popup
            .tapOutsideToDismiss(true)
            .dragGestureEnabled(true)
            .contentIgnoresSafeArea(true)
    }

    func createContent() -> some View {
        VStack(spacing: 15){
            TabView {
                Button(action: {
                    print("Text Print")
                    
                }, label: {
                    /*@START_MENU_TOKEN@*/Text("Button")/*@END_MENU_TOKEN@*/
                        .alignHorizontally(.center)
                        .background(Color.green)
                })
                Text("First")
                Text("Second")
                Text("Third")
                Text("Fourth")
            }
            .tabViewStyle(.page(indexDisplayMode: .never))
            .indexViewStyle(PageIndexViewStyle(backgroundDisplayMode: .never))
            .frame(height: 400)
            .background(Color.red)
        }
    }
}
```

**Comment by KavinduDissanayake** [comment]
@FulcrumOne
 // MARK: - Tapable Area
```swift
extension PopupStack {
    @ViewBuilder func createTapArea() -> some View { if tapOutsideClosesPopup {
        Color.black.opacity(0.44).ignoresSafeArea().onTapGesture(perform: items.last?.dismiss ?? {})
    }}
}
```
When I adding this for your library Issue Number 02 resolved(Stack Popup outside not dark color issue), can you check the tap issues and update you pod without I given code is it possible ?

**Comment by KavinduDissanayake** [comment]
@FulcrumOne  any update , sorry for disturbing

**Comment by FulcrumOne** [comment]
@KavinduDissanayake, sorry but I'm stuck at work. I'll try to update it this weekend

**Comment by KavinduDissanayake** [comment]
@FulcrumOne  Any Update ?

**Comment by KavinduDissanayake** [comment]
@FulcrumOne any update on this issue?

**Comment by FulcrumOne** [comment]
@KavinduDissanayake requested feature was implemented in version 2.3.0
